pkg/volume/plugins.go

```diff
@@ -84,6 +89,26 @@ type VolumeOptions struct {
 	Parameters map[string]string
 }
 
+// NodeResizeOptions contain options to be passed for node expansion.
+type NodeResizeOptions struct {
+	VolumeSpec *Spec
+
+	// DevicePath - location of actual device on the node. In case of CSI
+	// this just could be volumeID
+	DevicePath string
+
+	// DeviceMountPath location where device is mounted on the node. If volume type
+	// is attachable - this would be global mount path otherwise
+	// it would be location where volume was mounted for the pod
+	DeviceMountPath string
+
+	NewSize resource.Quantity
+	OldSize resource.Quantity
+
+	// CSIVolumePhase contains volume phase on the node
+	CSIVolumePhase CSIVolumePhaseType
```

Comment:
nit: this is leaking some CSI details. Can we use a name that's more representative of volume manager phases since this is technically a layer above CSI?

Reply:
I will fix this up in a follow up PR or in this PR itself it time permits.